backend/scripts/mirror_s3_data.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Mirror S3 data from a source deployment environment (usually production)
+# to a specified destination deployment environment (dev or staging)
+#
+# This will *add* S3 objects to the dest env, but will not remove existing objects (see NOTE, below).
+# The src S3 buckets will never be modified.
+
+set -e
+
+SCRIPTS_DIR=`dirname $0`
+. $SCRIPTS_DIR/set_src_dest_envs.sh
+
+echo Mirroring S3 data from $SRC_ENV to $DEST_ENV
+
+# NOTE: Add --delete if you want to clean up orphaned data. This would make the operation more destructive, of course!
+S3_SYNC_CMD="aws s3 sync --no-progress"
```

Comment:
If we arent including `--delete` as a default, how can we ensure that datasets removed from prod are also removed from staging/dev? They'll be orphaned/not available but we will still be storing the data which seems a little dangerous to me.

Reply:
It came down to data preservation/safety concerns, after I realized there were some unexpected data lying around in the buckets. I figured it's better to err on the side of caution in case some put some "extra" data in the dev or staging buckets for testing purposes. It's probably fine to add it in, particularly since these buckets are versioned. I just didn't want to cause additional problems before seeing that this mirroring was overall working.  I could make it a TODO instead of a NOTE? 😄